src/book/entities/Transaction.ts

```diff
@@ -75,6 +77,50 @@ export default class Transaction extends BaseEntity {
   })
   @v.Length(1, 2048)
     description!: string;
+
+  async save(options?: SaveOptions): Promise<this> {
+    const account = await super.save(options);
+
+    if (this.queryClient) {
+      updateCache({ queryClient: this.queryClient, entity: this });
+    }
+
+    return account;
+  }
+
+  async remove(options?: SaveOptions): Promise<this> {
+    if (this.queryClient) {
+      updateCache({ queryClient: this.queryClient, entity: this, isDelete: true });
+    }
+
+    const account = await super.remove(options);
+    return account;
+  }
+}
+
+/**
+ * Update some detail keys for consistency
+ *
+ * - /api/txs/latest -> Latest 5 transactions that happened
+ * - /api/txs/start -> The transaction date that happened the earliest
+ */
+export async function updateCache(
+  {
+    queryClient,
+  }: {
+    queryClient: QueryClient,
+    entity: Transaction,
+    isDelete?: boolean,
+  },
+) {
+  queryClient.invalidateQueries({
+    queryKey: [Transaction.CACHE_KEY, { name: 'latest' }],
+    refetchType: 'all',
```

Comment:
we need to set `all` because the queries may be inactive or active, we don't know here.